apps/docs-website/docs/core-philosophy/ruleset-design.md

```diff
@@ -7,25 +7,32 @@ description: 📐 What opinions should you expect from Sheriff
 
 ## No complex presets
 
-Sheriff doesn't offer “recommended” or “strict” presets. This config contains a predefined set of rules meant to act as guidelines for a **functional-light** programming style.
+Sheriff doesn’t offer “recommended” or “strict” presets. This config contains a predefined set of rules meant to act as guidelines for a **functional-light** programming style.
 
 ## No esoteric Typescript-only features
 
-Sheriff tries to be as faithful to Javascript as possible and conceives Typescript as a tool to enhance the capabilities that Javascript already has. Because of this, Sheriff discourages typescript-only features, like enums and overloads (and classes, and decorators by extensions, even though they also landed in Ecmascript).
+Sheriff tries to be as faithful to JavaScript as possible, and views TypeScript as a tool to enhance the capabilities that JavaScript already has. Because of this, Sheriff discourages TypeScript-only features, like enums and overloads (and classes, and decorators by extension, even though they also landed in ECMAScript).
```

Comment:
everything 👍
except for views > conceives

Reply:
I replaced conceives with views, not the other way around.

Reply:
yeah not but i mean that i like "conceives" better here

Reply:
Can you pick a different word?

> **Conceive**: form or devise (a plan or idea) in the mind[^1]

Sheriff didn't create TypeScript, it's _not_ conceiving it.

[^1]: Oxford

Reply:
Oh i get what you mean. 

Your wording is fine then 👍